src/internal/operators/groupBy.ts

```diff
@@ -143,49 +147,59 @@ export function groupBy<T, K, R>(
     const groupBySourceSubscriber = new GroupBySubscriber(
       subscriber,
       (value: T) => {
-        const key = keySelector(value);
+        // Because we have to notify all groups of any errors that occur in here,
+        // we have to add our own try/catch to ensure that those errors are propagated.
+        // OperatorSubscriber will only send the error to the main subscriber.
+        try {
```

Comment:
here, I added this try/catch to accommodate the fact that any error thrown in this particular `next` handler had to go to the `handleError` function, which previously was the function that we passed to `OperatorSubscriber`, before I changed the quirky behavior there.